### PR TITLE
Changing the Use Intune RBAC option requires GA role

### PR DESCRIPTION
**Pull request by PanuSaukko**
Hi,

I added info that you need to have a global admin role to change Use Intune RBAC option. You don't even see the flyout if you don't have the role.

Panu

### CODE REVIEW

File: memdocs/configmgr/cloud-attach/use-intune-rbac.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ To use Intune role-based access control for tenant attach rather than Configurat
 
 To enable Intune to manage user permissions for cloud-attached devices, use the following steps:  
 
-1. Open the [Microsoft Endpoint admin center](https://endpoint.microsoft.com) and sign in.
+1. Open the [Microsoft Endpoint admin center](https://endpoint.microsoft.com) and sign in as a user with the **Global Admin** role.
```
Comment (JohanFreelancer9) [inline memdocs/configmgr/cloud-attach/use-intune-rbac.md:62]:
```suggestion
1. Open the [Microsoft Endpoint admin center](https://endpoint.microsoft.com) and sign in as a **global admin**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @PanuSaukko, kindly commit the suggested change. Many thanks.

**Comment by mestew** [comment]
@PanuSaukko Hey Panu, I spoke to one of the devs on this. He noted that the specific required permission was Microsoft.Intune.Roles.Update. Can you verify that the role you normally use and that was denied access has the Roles\update permission? I'll toss you an email on this as well just in case you see something we aren't expecting.

File: memdocs/configmgr/cloud-attach/use-intune-rbac.md (hunk 1 above)
Comment (mestew) [reply memdocs/configmgr/cloud-attach/use-intune-rbac.md:62]: Change is obsolete. I am clarifying it with the specific permission that's needed.

Comment (mestew) [inline memdocs/configmgr/cloud-attach/use-intune-rbac.md:62]:
```suggestion
1. Open the [Microsoft Endpoint admin center](https://endpoint.microsoft.com) and sign in as a user that has the **Roles/Update** permission. For more information about the permission, see [custom role permissions in Intune](../../intune/fundamentals/create-custom-role.md).
```

**Comment by mestew** [comment]
@PanuSaukko thanks much for noticing that I was missed documenting the needed permission.

**Pull request merged**